### ISSUE TITLE
Multiple Home Support

### ISSUE THREAD

**Issue by bassrock** [KraigM/homebridge-nest#58]
Hi,

In my account I have multiple homes. One has just 1 camera while the other has a camera, nest and nest protect. When I load home bridge, only the home with the camera has its devices show up in my setup.

**Comment by KraigM** [comment]
Never tried it with multiple homes.  From what I can tell it should be showing you all the devices from all Nest homes.  We prob should build some more functionality for the multiple homes case but not sure why you other devices are not showing up

**Comment by bassrock** [comment]
Yep only one of the homes devices is showing up and its the first home that was on my account, then I was added to another home. It could be the fact that I was added to another home instead of creating my own. Willing to help debug the issue however I can to.

**Comment by KraigM** [comment]
OK this version should output the raw data we currently get from Nest

```
npm install -g github:kraigm/homebridge-nest#e00c92b67926e6f6474a1671c0650ffd79f875e3
```

**Comment by bassrock** [comment]
@KraigM Didn't seem to add any extra logging: just have:

```
[6/14/2016, 11:40:07 AM] Software version for Office Camera (Office) is: 205-600052
[6/14/2016, 11:40:07 AM] Away for Office Camera (Office) is: 0
[6/14/2016, 11:40:07 AM] Initializing platform accessory 'Office Camera (Office)'...
Scan this code with your HomeKit App on your iOS device to pair with Homebridge:
```

**Comment by KraigM** [comment]
Sorry you need to start homebridge with the `-D` option

**Comment by bassrock** [comment]
@KraigM Ok so looking at the data it outputs it only lists the one camera from the original home. Not the invited home that has the nest and nest protect.   The data seems to have some specific info for me so don't really want to post it here.

**Comment by KraigM** [comment]
Ok I was just curious if it was hiding the linked homes in a special field, but I guess it isn't...

**Comment by bassrock** [comment]
Nope not seeing any of that. Looks like data specific to one home no link to another.

**Comment by KraigM** [comment]
Do you have access to the other account?

**Comment by KraigM** [comment]
So you could just use that account instead

**Comment by wootwoot1234** [comment]
I'm actually have the exact opposite problem but I think it's related.  I have two homes (one is a vacation rental) each with a nest thermostat.  I don't want to control the thermostat in the vacation rental but when I say something like "Set the temperature to 75 degrees" it changes both.  So I have two homes setup in home.nest.com and both thermostats are showing up on homebridge.  Both of my devices are on the same wireless network so maybe that has something to do with it finding both homes.  Anyway, is it possible to remove one home?

I can start a different thread if this isn't the correct place to ask this, just let me know.
